src/data/dailylimits.txt

```diff
@@ -185,6 +185,7 @@ Drink	cup of "tea"	_speakeasyDrinksDrunk	3
 Drink	Doc Clock's thyme cocktail	_docClocksThymeCocktailDrunk
 Drink	Flivver	_speakeasyDrinksDrunk	3
 Drink	Frosty's frosty mug	_frostyMugUsed
+Drink	Hodgman's blanket	_hodgmansBlanketDrunk
```

Comment:
there's also tin cup of mulligan stew, which I had in #2151.

Either that one can go in, and you can make this PR just about the fork / mug, or you can add it here. Either or.

Reply:
Probably easier to send that one in and remove it from here -- I hope that one will go in easier.

Reply:
Yours should go in for the Hodgman stuff, especially since you remembered both items. I'll (figure out how to) remove it from mine and reduce this to the checks for failed fork and mug uses. Thanks for the help focusing this one better.

Reply:
alright, that one's gone in, so if you rebase yours you should be able to remove the blanket stuff.

You could also just put the fork / mug stuff in here, then I can approve and merge because I think that's fine, and then we can do the pickle juice overdrink thing as a follow-up to that?

Reply:
I'm going to go ahead and close this one. It'll be a lot cleaner that way, since I can't figure out how to just rebase selections of my original branch in a way that makes any kind of sense while leaving the branch existing the whole time. I'll open another PR with the mug & fork bits this afternoon and we can work out how to handle the prioritize the limits situation after that.

Also, are you already working on chest mimic support, or do you know if someone else is?

Reply:
I don't know of anyone working on the mimic.

Parsing the egg description should wait until CDM adds ids (to make that a lot easier), but things like the preferences for use can go in earlier.